packages/components/src/Chips/Chips.tsx

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import { useCloseIconProps } from './Chips.hooks.js'
+import { useChipsStyles } from './Chips.styles.js'
+import type { ChipsProps } from './Chips.types.js'
+import { Icon, IconButton } from '@fluentui/react'
+
+export function Chips({
+	items,
+	styles,
+	onClose,
+	onClick,
+}: ChipsProps): JSX.Element {
+	const _styles = useChipsStyles(styles)
+	const closeProps = useCloseIconProps(_styles)
+	return (
+		<div style={_styles.root}>
+			{items.map((item) => {
```

Comment:
Should we memoize this?

Reply:
I guess, maybe not if you don't want "items" specifically to trigger a rerender, but the items contained within it.

Reply:
I've always thought new functions/objects were ok within the map since the outer list would need to be mutated to trigger re-render. However, I think I'd like to pull Chip out into an independent component anyway.

Reply:
I'm probably wrong, but I think that is only if you `React.memo` the `Chips` component, no?  I think `Chips` will get run anytime the parent component decides to re-render `Chips`, regardless of whether or not the props change for `Chips`.  Maybe React memoizes the components based on the props by default now ¯\_(ツ)_/¯?